sdk/tables/azure-data-tables/tests/test_table_client_cosmos.py

```diff
@@ -627,25 +863,3 @@ def test_validate_cosmos_tablename(self):
             _validate_cosmos_tablename("a ")
         with pytest.raises(ValueError):
             _validate_cosmos_tablename("a" * 255)
-
-    def test_create_service_with_connection_string_cosmos(self):
```

Comment:
We don't need this any more?

Reply:
I find this is a duplicate test, there's a same one on line 342(left side). Same for the removed test in async file.